cookbook/configuration/pdo_session_storage.rst

```diff
@@ -217,3 +217,85 @@ For MSSQL, the statement might look like the following:
             ALLOW_PAGE_LOCKS  = ON
         ) ON [PRIMARY]
     ) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
+
+Doctrine DBAL Session Storage
+-----------------------------
+
+Symfony applications can also use a Doctrine DBAL based session storage. This
+alternative implementation is very similar to the ``PdoSessionHandler`` explained
+above, but uses a Doctrine connection and thus also works with non-PDO-based
+drivers like mysqli and OCI8.
+
+The only significant disadvantage of the Doctrine DBAL session storage comparing
```

Comment:
note that these paragraphs are true for 2.3, but not for 2.6+ (the DBAL session storage has not been migrated to implement session locking yet AFAIK)